package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "misty-config",
+  "version": "1.0.0",
+  "description": "Cascading configuration with environment variable support",
+  "main": "index.js",
+  "scripts": {
+  },
+  "author": "Body Labs Inc.",
+  "license": "MPL-2.0",
```

Comment:
My bad. I copied the line from `rho-cc-callback`, which of course is an exception. Fixed